src/blocks/HeaderBlockBrandPivot/index.tsx

```diff
@@ -207,41 +217,135 @@ export const Header: React.FC<{ story: GlobalStory } & HeaderBlockProps> = (
             />
             <ContentWrapper>
               <InnerHeaderWrapper>
-                <RightContainer>
+                <LeftContainer>
                   <Burger
                     isOpen={isOpen}
                     isClosing={isClosing}
                     onClick={toggleOpen}
                     preventInverse={props.inverse_colors && isOpen}
                   />
 
+                  <MediaQuery query="(min-width: 1001px)">
+                    <ContextContainer>
+                      {(context) => (
+                        <LogoLink
+                          href={
+                            '/' + (context.lang === 'sv' ? '' : context.lang)
+                          }
+                        >
+                          <HedvigLogotype width={98} />
+                        </LogoLink>
+                      )}
+                    </ContextContainer>
+                  </MediaQuery>
+                </LeftContainer>
+
+                <MediaQuery query="(max-width: 1000px)">
                   <ContextContainer>
                     {(context) => (
-                      <LogoLink
+                      <Wordmark
                         href={'/' + (context.lang === 'sv' ? '' : context.lang)}
                       >
-                        <HedvigLogotype width={98} />
-                      </LogoLink>
+                        <HedvigH />
+                      </Wordmark>
                     )}
                   </ContextContainer>
-                </RightContainer>
+                </MediaQuery>
 
-                {!isOpen && (
-                  <>
+                <Menu open={isOpen}>
+                  {(props.story.content.header_menu_items ?? []).map(
+                    (menuItem) => (
+                      <MenuItem menuItem={menuItem} key={menuItem._uid} />
+                    ),
+                  )}
+
+                  <MediaQuery query="(max-width: 1000px)">
```

Comment:
Ah, I realized now that it's the `<MediaQuery>` that causes the flickering. So if we want to show/hide depending on viewport we should wrap it in a div and just use css media querys to get rid of that problem :)

Reply:
Great 👍